### PR TITLE
feat: extract entities mapping from sputnik http service class

### PR DESCRIPTION
**Pull request by stee1house**

### CODE REVIEW

File: services/HttpService/HttpService.ts [hunk 1]
```diff
@@ -11,11 +35,95 @@ export class HttpService {
...
+      if (!responseConfig.responseMapper) {
+        return response;
+      }
+
+      switch (responseConfig.responseMapper.name) {
+        case API_MAPPERS.mapDaoDTOtoDao:
+          response.data = mapDaoDTOtoDao(response.data);
+          break;
+        case API_MAPPERS.mapDaoFeedItemResponseToDaoFeedItemList:
+          if (Array.isArray(response.data.data)) {
```
Comment (AlexeyPonomarenko) [inline services/HttpService/HttpService.ts:51]: Can we do this check (and all similar checks from this file) and apply proper processing in mapper?

File: constants/mappers.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export const API_MAPPERS = {
+  mapDaoDTOtoDao: 'mapDaoDTOtoDao',
```
Comment (AlexeyPonomarenko) [inline constants/mappers.ts]: I consider we should write inner fields of constants in SCREAMING_SNAKE_CASE. @VERZUOL1 @karpovdmitry WDYT?

File: services/HttpService/HttpService.ts (hunk 1 above)
Comment (stee1house) [reply services/HttpService/HttpService.ts:51]: I think the mapper function shouldn't decide by incoming data what it should return. Maybe a better solution is if we create a separate type for each condition. What do you think?

File: constants/mappers.ts (hunk 2 above)
Comment (VERZUOL1) [reply constants/mappers.ts]: Agree. This is the format we use across the app.

Comment (stee1house) [reply constants/mappers.ts]: @VERZUOL1 @AlexeyPonomarenko fixed

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**